src/main/java/com/zaxxer/sansorm/internal/OrmReader.java

```diff
@@ -173,7 +172,7 @@ public static ResultSet statementToResultSet(PreparedStatement stmt, Object... a
 
       Introspected introspected = Introspector.getIntrospected(target.getClass());
       for (int column = metaData.getColumnCount(); column > 0; column--) {
-         String columnName = metaData.getColumnName(column).toLowerCase();
```

Comment:
toLowerCase(): No longer needed because Introspected.getFieldColumnInfo() is case insensitive.